component/component/vshn_services.jsonnet

```diff
@@ -30,14 +30,29 @@ local maintenanceRole = kube.ClusterRole(maintenanceRoleName) {
   ],
 };
 
+
 local maintenanceClusterRoleBinding = kube.ClusterRoleBinding('crossplane:appcat:job:helm:maintenance') + {
   roleRef_: maintenanceRole,
   subjects_: [ maintenanceServiceAccount ],
 };
 
+// this role is necessary for maintenance logic to get info if vacuum or repack is enabled in .spec.parameters.service
+// clusterrolebinding is created as object and grant permissions to maintenance service account in instance namespace
+// don't add LIST verb here, maintenance service account could in some specific cases be reused by customers i shared clusters
```

Comment:
We shouldn't really add any other verbs here. If we ever need more permissions, I would strongly suggest to move to the appcat-contol namespace, like we do with the helmpatcher.

The `get` verb should be fine, there's not much a customer can do with it. Although guessing names for claims might not be too hard, as they don't have any random suffixes.

What do you think @zugao